### PR TITLE
Change system branding

### LINKED ISSUE #103: Change system branding
* Change name to "eventyay"
* Use same logo as on eventyay-talk
* Change to "powered by eventyay" with link to eventyay.com
* Change the name "pretix" in all text fields to "the system" or similar. It should make sense.
* In Readme.md change from Pretix to Eventyay-Tickets

Compare to https://github.com/fossasia/eventyay-talk/pull/69/files

Screenshot from 2024-05-22 18-57-10

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #103 

image
image

### CODE REVIEW

**Review by mariobehling — CHANGES_REQUESTED** [review]
Good job as a start, but looks like you used the favicon on the top left. In any case it is too small. Please check more through your changes and see if you catch more those issues. Thanks

Current
Screenshot from 2024-05-24 08-22-14

Expected
Screenshot from 2024-05-24 08-24-10

**Comment by mariobehling** [comment]
Ahh I see. I think because the background color is blue, the image does not appear properly. Could you please use an image with colors. Thanks.

**Comment by progmatic-99** [comment]
@mariobehling ok, doing that

**Comment by progmatic-99** [comment]
image
image

**Comment by mariobehling** [comment]
> image 

Could you change the background color to the same color as eventyay-talk, please?

**Comment by progmatic-99** [comment]
ok @mariobehling

File: src/pretix/control/templates/pretixcontrol/item/index.html [hunk 1]
```diff
@@ -1,146 +1,169 @@
...
-                        </div>
-                        {% bootstrap_field form.category layout="control" %}
+{% extends "pretixcontrol/item/base.html" %} {% load i18n %} {% load bootstrap3
+%} {% load formset_tags %} {% block inside %}
+<form
+  action=""
+  method="post"
+  class="form-horizontal"
+  enctype="multipart/form-data"
+>
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/item/index.html]: Can you please revert these changes, they are unrelated. You probably have your linter/ruff set to 80 char line length

File: src/pretix/control/templates/pretixcontrol/item/index.html [hunk 2]
```diff
@@ -1,146 +1,169 @@
...
+  {% csrf_token %}
+  <div class="row">
+    <div class="col-xs-12 col-lg-10">
+      <div class="tabbed-form">
+        <fieldset>
+          <legend>{% trans "General" %}</legend>
+          {% bootstrap_field form.active layout="control" %} {% bootstrap_field
+          form.name layout="control" %}
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/item/index.html]: Same, that is unrelated change

File: src/pretix/control/templates/pretixcontrol/item/index.html [hunk 3]
```diff
@@ -1,146 +1,169 @@
...
+                    type="radio"
+                    value="on"
+                    name="{{ form.admission.html_name }}"
+                    {%
+                    if
+                    form.admission.value
+                    %}checked{%
+                    endif
+                    %}
+                  />
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/item/index.html]: Same

File: src/pretix/control/templates/pretixcontrol/item/index.html [hunk 4]
```diff
@@ -1,146 +1,169 @@
...
+                  <span class="fa fa-fw fa-user"></span>
+                  <strong>{% trans "Admission product" %}</strong><br />
+                  <div class="help-block">
+                    {% blocktrans trimmed %} Every purchase of this product
+                    represents one person who is allowed to enter your event. By
+                    default, eventyay will only ask for attendee information and
+                    offer ticket downloads for these products. {% endblocktrans
+                    %}
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/item/index.html]: Same

File: src/pretix/control/templates/pretixcontrol/item/index.html [hunk 5]
```diff
@@ -1,146 +1,169 @@
...
+                  </div>
+                  <div class="help-block">
+                    {% blocktrans trimmed %} This option should be set for most
+                    things that you would call a "ticket". For product add-ons
+                    or bundles, this should be set on the main ticket, except if
+                    the add-on products or bundled products represent additional
+                    people (e.g. group bundles). {% endblocktrans %}
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/item/index.html]: Same

File: src/pretix/control/templates/pretixcontrol/item/index.html [hunk 6]
```diff
@@ -1,146 +1,169 @@
...
+      </div>
+    </div>
+    <div class="col-xs-12 col-lg-2">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">{% trans "Product history" %}</h3>
         </div>
-    </form>
+        {% include "pretixcontrol/includes/logs.html" with obj=item %}
+      </div>
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/item/index.html]: All of the changes in this file should be reverted, I don't think there is anything related to eventyay branding included here.

File: src/pretix/control/templates/pretixcontrol/organizers/delete.html [hunk 7]
```diff
@@ -1,45 +1,42 @@
...
+<p>
+  {% trans "This organizer account can not be deleted as it already contains
+  orders, invoices, or devices." %}
+</p>
+<p>
+  {% blocktrans trimmed %} eventyay does not allow deleting orders once they
+  have been placed in order to be audit-proof and trustable by financial
+  authorities. {% endblocktrans %}
+</p>
+{% endif %} {% endblock %}
```
Comment (norbusan) [inline src/pretix/control/templates/pretixcontrol/organizers/delete.html:42]: Same as the above, please don't include unrelated changes

File: src/pretix/static/pretixbase/img/pretix-logo.svg
Comment (norbusan) [inline src/pretix/static/pretixbase/img/pretix-logo.svg:1]: The size of the images don't match, is this a problem?

**Review by norbusan — CHANGES_REQUESTED** [review]

File: src/pretix/static/pretixbase/img/pretix-icon-colored.svg
Comment (mariobehling) [inline src/pretix/static/pretixbase/img/pretix-icon-colored.svg:1]: Please make it same 150px x 150px even though it is a svg. If possible crop to the actual image.

File: src/pretix/static/pretixbase/img/pretix-icon.svg
Comment (mariobehling) [inline src/pretix/static/pretixbase/img/pretix-icon.svg:1]: 150px x150 px and crop to actual image, please.

File: src/pretix/static/pretixbase/img/pretix-logo.svg
Comment (mariobehling) [inline src/pretix/static/pretixbase/img/pretix-logo.svg:1]: Please W: 254px x H: 110px

**Comment by progmatic-99** [comment]
@AviGawande i don't see your changes

**Comment by AviGawande** [comment]
@progmatic-99  raise pr to your branch.

File: src/pretix/static/pretixbase/img/pretix-icon-colored.svg
Comment (mariobehling) [inline src/pretix/static/pretixbase/img/pretix-icon-colored.svg:1]: Please rename pretix to eventyay in the image file.

File: src/pretix/static/pretixbase/img/pretix-logo.svg
Comment (mariobehling) [inline src/pretix/static/pretixbase/img/pretix-logo.svg:1]: Please rename pretix to eventyay in the image file.

File: src/pretix/static/pretixbase/img/pretix-icon.svg
Comment (mariobehling) [inline src/pretix/static/pretixbase/img/pretix-icon.svg:1]: Please rename pretix to eventyay in all image files.

File: src/pretix/presale/templates/pretixpresale/base_footer.html [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
 {% comment %}
 
     Hey, you!
 
-    Yes, you found the place in which you can remove or change the link to pretix from the shop footer.
+    Yes, you found the place in which you can remove or change the link to eventyay from the shop footer.
     However, I want to politely ask you to re-consider and do not remove it.
 
-    Creating pretix has been an huge effort that took many years and maintaining it is still a lot of
+    Creating eventyay has been an huge effort that took many years and maintaining it is still a lot of
```
Comment (mariobehling) [inline src/pretix/presale/templates/pretixpresale/base_footer.html]: Please delete this paragraph and the email.

File: src/pretix/presale/templates/pretixpresale/index.html [hunk 9]
```diff
@@ -6,8 +6,8 @@
     <div class="form-signin" action="" method="post">
         <h3>{% trans "Hello!" %}</h3>
         <p>
-            {% blocktrans trimmed with a_attr="href='http://pretix.eu'" %}
-                This is a self-hosted installation of <a {{ a_attr }}>pretix, your free and open source ticket sales
+            {% blocktrans trimmed with a_attr="href='https://eventyay.com'" %}
```
Comment (mariobehling) [inline src/pretix/presale/templates/pretixpresale/index.html:9]: Change to:  This is a self-hosted installation of <a {{ a_attr }}>eventyay, your open source event solution.

File: src/pretix/plugins/returnurl/templates/returnurl/settings.html
Comment (mariobehling) [inline src/pretix/plugins/returnurl/templates/returnurl/settings.html:1]: This feature is useful if you use only the payment component of the system but build your own checkout interface for other steps.

File: src/pretix/control/templates/pretixcontrol/user/2fa_enable.html [hunk 10]
```diff
@@ -10,7 +10,7 @@ <h1>{% trans "Enable two-factor authentication" %}</h1>
             {% trans "Do you really want to enable two-factor authentication?" %}
         </p>
         <p>
-            {% trans "You will no longer be able to log in to pretix without one of your configured devices." %}
+            {% trans "You will no longer be able to log in to eventyay without one of your configured devices." %}
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/user/2fa_enable.html]: Change to: You will no longer be able to log in without one of your configured devices.

File: src/pretix/control/templates/pretixcontrol/multi_languages_widget.html [hunk 11]
```diff
@@ -10,7 +10,7 @@
             {% trans "Translation in development" %}
         </span>
     {% elif not widget.official %}
-        <span class="label label-warning" data-toggle="tooltip" title="{% trans "This translation is not maintained by the pretix team. We cannot vouch for its correctness and new or recently changed features might not be translated and will show in English instead. You can help translating at translate.pretix.eu." %}">
+        <span class="label label-warning" data-toggle="tooltip" title="{% trans "This translation is not maintained by the eventyay team. We cannot vouch for its correctness and new or recently changed features might not be translated and will show in English instead. You can help translating at translate.pretix.eu." %}">
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/multi_languages_widget.html]: This translation is community-maintained and requires further verification by additional translators. New or recently updated features may not be translated yet and will appear in English.

File: src/pretix/control/templates/pretixcontrol/items/quotas.html [hunk 12]
```diff
@@ -7,7 +7,7 @@ <h1>{% trans "Quotas" %}</h1>
     <p>
         {% blocktrans trimmed %}
             To make your products actually available, you also need quotas. Quotas define, how many instances of
-            your product pretix will sell. This way, you can configure whether your event can take an unlimited
+            your product eventyay will sell. This way, you can configure whether your event can take an unlimited
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/items/quotas.html]: Change the entire paragraph as follows: 

To ensure your products are available as intended, you need to set quotas. Quotas determine how many instances of your product can be sold on the platform. This allows you to configure whether your event can accommodate an unlimited number of attendees or if the number of attendees is capped. You can assign a product to multiple quotas to meet more complex requirements, such as limiting the total number of tickets sold while also restricting the number of specific ticket types simultaneously.

File: doc/_templates/index.html [hunk 13]
```diff
@@ -21,7 +21,7 @@ <h2>Documentation structure</h2>
             <a href="user/index.html">
                 <strong>User Guide</strong>
             </a>
-            <p>Go here to find information on how to configure and use pretix as an event organizer.</p>
+            <p>Go here to find information on how to configure and use eventyay as an event organizer.</p>
```
Comment (mariobehling) [inline doc/_templates/index.html]: Go here to find information on how to configure and use the platform as an event organizer.

File: doc/_templates/index.html [hunk 14]
```diff
@@ -79,7 +79,7 @@ <h2>Documentation structure</h2>
             <a href="plugins/index.html">
                 <strong>Plugin docs</strong>
             </a>
-            <p>Documentation and details on plugins that ship with pretix or are officially supported.</p>
+            <p>Documentation and details on plugins that ship with eventyay or are officially supported.</p>
```
Comment (mariobehling) [inline doc/_templates/index.html]: Documentation and details on plugins that ship with the platform or are officially supported.

File: doc/_templates/index.html [hunk 15]
```diff
@@ -109,7 +109,7 @@ <h2>Useful links</h2>
             <a href="https://pretix.eu" target="_blank">
                 <strong>Project website</strong>
             </a>
-            <p>pretix.eu is the central entry-point to the pretix project and also the home of the commercial hosting
+            <p>pretix.eu is the central entry-point to the eventyay project and also the home of the commercial hosting
```
Comment (mariobehling) [inline doc/_templates/index.html]: eventyay.com is your gateway to the eventyay project, offering premier commercial hosting services all in one place.

File: doc/_templates/index.html [hunk 16]
```diff
@@ -123,7 +123,7 @@ <h2>Useful links</h2>
             <a href="https://github.com/pretix/pretix" target="_blank">
                 <strong>GitHub repository</strong>
             </a>
-            <p>Our main source code repository contains all code that is part of pretix as well as some plugins and the
+            <p>Our main source code repository contains all code that is part of eventyay as well as some plugins and the
```
Comment (mariobehling) [inline doc/_templates/index.html]: The source code repository contains the code that is part of eventyay tickets as well as other components, plugins and the

File: doc/_templates/index.html [hunk 17]
```diff
@@ -138,8 +138,8 @@ <h2>Useful links</h2>
             <a href="https://pretix.eu/about/en/blog/" target="_blank">
                 <strong>Project blog</strong>
             </a>
-            <p>This important information source contains all release notes for all stable releases of pretix as well as
-                general news on pretix as a project.</p>
+            <p>This important information source contains all release notes for all stable releases of eventyay as well as
```
Comment (mariobehling) [inline doc/_templates/index.html]: This essential resource provides all release notes for stable eventyay releases, along with general news and updates.

File: doc/admin/config.rst [hunk 18]
```diff
@@ -37,15 +37,15 @@ pretix settings
 Example::
 
     [pretix]
-    instance_name=pretix.de
+    instance_name=eventyay
     url=http://localhost
     currency=EUR
     datadir=/data
     plugins_default=pretix.plugins.sendmail,pretix.plugins.statistics
```
Comment (mariobehling) [inline doc/admin/config.rst:44]: What about changing pretix here? If this is not easily possible please open an issue describing what needs to be done to change "pretix" here.

File: src/pretix/control/templates/pretixcontrol/auth/oauth_authorization.html [hunk 19]
```diff
@@ -17,7 +17,7 @@ <h3>{% trans "Authorize an application" %}</h3>
             <p>
                 {% blocktrans trimmed with application=application.name %}
                     Do you really want to grant the application <strong>{{ application }}</strong> access to your
-                    pretix account?
+                    eventyay account?
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/auth/oauth_authorization.html]: Do you really want to grant the application access to your account?

File: src/pretix/control/templates/pretixcontrol/auth/oauth_authorization.html [hunk 20]
```diff
@@ -39,8 +39,8 @@ <h3>{% trans "Authorize an application" %}</h3>
             {% bootstrap_form_errors form layout="control" %}
             <p class="text-danger">
                 {% blocktrans trimmed %}
-                    This application has <strong>not</strong> been reviewed by the pretix team. Granting access to your
-                    pretix account happens at your own risk.
+                    This application has <strong>not</strong> been reviewed by the eventyay team. Granting access to your
+                    eventyay account happens at your own risk.
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/auth/oauth_authorization.html]: This application has been reviewed, but granting access to your account is at your own risk.

File: src/pretix/control/templates/pretixcontrol/base.html [hunk 21]
```diff
@@ -385,7 +385,7 @@
                         <div class="alert alert-info">
                             <a href="{% url "control:global.update" %}">
                                 {% blocktrans trimmed %}
-                                    Starting with version 1.2.0, pretix automatically checks for updates in the background.
+                                    Starting with version 1.2.0, system automatically checks for updates in the background.
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/base.html]: The system automatically checks for updates in the background.

File: src/pretix/control/templates/pretixcontrol/event/delete.html [hunk 22]
```diff
@@ -37,7 +37,7 @@ <h1>{% trans "Delete event" %}</h1>
         </p>
         <p>
             {% blocktrans trimmed %}
-                pretix does not allow deleting orders once they have been placed in order to be audit-proof and
+                eventyay does not allow deleting orders once they have been placed in order to be audit-proof and
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/event/delete.html]: To ensure audit compliance and maintain trust with financial authorities, deleting orders once they have been placed is not permitted.

File: src/pretix/control/templates/pretixcontrol/event/widget.html [hunk 23]
```diff
@@ -8,7 +8,7 @@
     <h1>{% trans "Widget" %}</h1>
     <p>
         {% blocktrans trimmed %}
-            The pretix widget is a way to embed your ticket shop into your event website. This way, your visitors can
+            The eventyay widget is a way to embed your ticket shop into your event website. This way, your visitors can
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/event/widget.html]: A widget allows you to embed your ticket shop directly into your event website. This enables your visitors to purchase tickets instantly without leaving your site.

File: src/pretix/control/templates/pretixcontrol/item/create.html [hunk 24]
```diff
@@ -26,7 +26,7 @@
                             <div class="help-block">
                                 {% blocktrans trimmed %}
                                     Every purchase of this product represents one person who is allowed to enter your event.
-                                    By default, pretix will only ask for attendee information and offer ticket downloads for these products.
+                                    By default, eventyay will only ask for attendee information and offer ticket downloads for these products.
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/item/create.html]: By default, the system will request attendee information and provide ticket downloads for these products.

File: src/pretix/control/templates/pretixcontrol/item/create.html [hunk 25]
```diff
@@ -44,7 +44,7 @@
                             <strong>{% trans "Non-admission product" %}</strong>
                             <div class="help-block">
                                 {% blocktrans trimmed %}
-                                    A product that does not represent a person. By default, pretix will not ask for attendee information or offer
+                                    A product that does not represent a person. By default, eventyay will not ask for attendee information or offer
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/item/create.html]: For products that do not require a person to be present, such as a T-shirt, the system will, by default, not request attendee information or provide ticket downloads.

File: src/pretix/control/templates/pretixcontrol/organizers/team_members.html [hunk 26]
```diff
@@ -72,7 +72,7 @@ <h3>{% trans "Team members" %}</h3>
                     {% bootstrap_field add_form.user layout='inline' %}<br>
                     {% blocktrans trimmed %}
                         To add a new user, you can enter their email address here. If they already have a
```
Comment (mariobehling) [inline src/pretix/control/templates/pretixcontrol/organizers/team_members.html]: If they already have a -> If they already have an

File: src/pretix/settings.py [hunk 27]
```diff
@@ -116,7 +116,7 @@
 
 MEDIA_URL = config.get('urls', 'media', fallback='/media/')
 
-PRETIX_INSTANCE_NAME = config.get('pretix', 'instance_name', fallback='pretix.de')
+PRETIX_INSTANCE_NAME = config.get('pretix', 'instance_name', fallback='eventyay')
```
Comment (mariobehling) [inline src/pretix/settings.py]: Can this be changed from "PRETIX_INSTANCE_NAME" to "NSTANCE_NAME" and the following items as well?

If this is a big complicated please open a separate issue about this.

**Review by mariobehling — CHANGES_REQUESTED** [review]

File: doc/development/api/email.rst [hunk 28]
```diff
@@ -87,7 +87,7 @@ To use it, you just need to implement some variables:
 The template is passed the following context variables:
 
 ``site``
-   Name of the pretix installation (``settings.PRETIX_INSTANCE_NAME``)
+   Name of the pretix installation (``settings.INSTANCE_NAME``)
```
Comment (mariobehling) [inline doc/development/api/email.rst]: change to "eventyay tickets installation"

File: src/pretix/testutils/settings.py [hunk 29]
```diff
@@ -21,7 +21,7 @@
 COMPRESS_ENABLED = COMPRESS_OFFLINE = False
 COMPRESS_CACHE_BACKEND = 'testcache'
 STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
-PRETIX_INSTANCE_NAME = 'pretix.eu'
+INSTANCE_NAME = 'pretix.eu'
```
Comment (mariobehling) [inline src/pretix/testutils/settings.py]: eventyay.com

File: src/tests/presale/test_event.py [hunk 30]
```diff
@@ -1153,7 +1153,7 @@ def test_timezone_header_footer(self):
     def test_metadata(self):
         ical = self.client.get('/%s/%s/ical/' % (self.orga.slug, self.event.slug)).content.decode()
         self.assertIn('VERSION:2.0', ical, 'incorrect version tag - 2.0')
-        self.assertIn('-//pretix//%s//' % settings.PRETIX_INSTANCE_NAME, ical, 'incorrect PRODID')
+        self.assertIn('-//pretix//%s//' % settings.INSTANCE_NAME, ical, 'incorrect PRODID')
```
Comment (mariobehling) [inline src/tests/presale/test_event.py]: Can this be renamed to "eventyay" ?

File: doc/development/api/email.rst (hunk 28 above)
Comment (progmatic-99) [reply doc/development/api/email.rst]: done

**Comment by mariobehling** [comment]
Ok, the Django 21 PR has been finally merged. Need to test if this PR works after those changes. Looks good to me in general.

**Review by mariobehling — APPROVED** [review]

**Pull request merged**